app/assets/javascripts/ui_models/application.ts

```diff
@@ -27,11 +28,12 @@ import {
 import { AppState } from '@/ui_models/app_state';
 import { SNWebCrypto } from 'sncrypto/dist/sncrypto-web';
 import { Bridge } from '@/services/bridge';
+import { DeinitSource } from 'snjs/dist/@types/types';
 
 type WebServices = {
   appState: AppState
   desktopService: DesktopManager
-  lockService: LockManager
+  autolockService: AutolockService
```

Comment:
Do you prefer the term Service to Manager? :) (not an issue, just curious)

Reply:
The distinction I decided on after a lot of self-quarreling was:

- a manager manages things for you and runs things without you being aware of them, like registering for an observer and responding to changes
- a service centers more on how other parts of the code can use it to do what they want

Obviously there will be a lot of overlap so it's just a guideline. But I suppose that by these guidelines the autolock service is more of a manager than a service.